Can now compare changes (and other bug fixes)

mallchin opened the pull request:
NEW: Can now compare changes

FIX: Fixed detecting proxy settings
FIX: Fixed configuration descriptions
FIX: Removed references to HipChat
FIX: Removed Composer (wasn't used?!?)

mallchin committed:
Update plugin.info.txt

mallchin committed:
Update default.php

mallchin committed:
Update metadata.php

mallchin committed:
Update settings.php

mallchin committed:
Delete userscript.js

mallchin committed:
Update action.php

mallchin committed:
Update action.php

mallchin committed:
Initial commit

mallchin committed:
Update README.md

littleiffel commented:
Okay this goes beyond my knowledge of DokuWiki, I trust that you know what you are doing :)

mallchin commented:
It should be all good :) I fixed a few things, renamed the plugin base directory, removed all unused code, and included the changelog include to query revisions.  I might do some more development tomorrow.

littleiffel commented:
Excellent. Thanks for improving the plugin
